csit-sut/Dockerfile

```diff
@@ -116,13 +114,22 @@ RUN pip install \
         scp==0.10.2 \
         six==1.12.0 \
         dill==0.2.8.2 \
-        numpy==1.14.5 \
-        scipy==1.1.0
+        numpy==1.14.5
+
+# ARM workaround
+RUN pip install scipy==1.1.0
 
 # VPP PIP pre-cache
 RUN pip install \
         aenum
 
+# Kernel image for VM tests download and extract
+COPY get-linux-vm-image.sh /root/get-linux-vm-image.sh
+RUN chmod 755 /root/get-linux-vm-image.sh
+RUN /root/get-linux-vm-image.sh
+
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
Must be part of /root/get-linux-vm-image.sh script + should be also on line 40 of single apt-get

Reply:
The problem is that this command has no effect. Every "RUN" is creating new Docker layer, if you do not make it clean in that layer, you lost access to clean it and resulting file will be huge as each apt-get is downloading lots of data.